server/gfmd/db_pgsql.c

```diff
@@ -4361,4 +4496,9 @@ const struct db_ops db_pgsql_ops = {
 	gfarm_pgsql_mdhost_load,
 
 	gfarm_pgsql_fsngroup_modify,
+
```

Comment:
move the user_auth related functions to just after the gfarm_pgsql_user_load() function,
to match the modified order of struct db_ops.

Reply:
commit [429eb3d](https://github.com/oss-tsukuba/gfarm/pull/1141/commits/429eb3daa151165a09117234d565149a2a1ac4e6) has not applied above change.
Could you change it?